content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -243,7 +243,7 @@ ARCH="amd64"
 curl -L "https://github.com/kubernetes-sigs/cri-tools/releases/download/${CRICTL_VERSION}/crictl-${CRICTL_VERSION}-linux-${ARCH}.tar.gz" | sudo tar -C $DOWNLOAD_DIR -xz
 ```
 
-Install `kubeadm`, `kubelet`, `kubectl` and add a `kubelet` systemd service:
+Install `kubeadm`, `kubelet` and add a `kubelet` systemd service:
```

Comment:
line 261 tells how to install kubectl separately, so the old list of components seems correct here.

Reply:
well I think the bash scripts started at line 248  is supporting line 246, line 261 is the next step .
the line 246 make ones to think kubectl has been installed through the scripts.

Reply:
i think you are making a good point. let's leave it to the pr approvers to decide.

Reply:
I think the current description is acceptable.
The operations that follow this line include the installation of the binaries and the creation of the systemd service.
You can still challenge that why start the `kubelet` service after downloading `kubectl`. Well, sometimes the order doesn't matter that much. The irrelevance is implied in the operations themselves.